openreview/conference/webfield.py

```diff
@@ -204,32 +204,34 @@ def set_bid_page(self, conference, invitation, group_id, request_count, instruct
             invitation.web = content
             return self.client.post_invitation(invitation)
 
-    def set_recommendation_page(self, conference, invitation):
+    def set_recommendation_page(self, conference, invitation, assignment_title, score_ids):
 
         default_header = {
-            'title': conference.get_short_name() + ' Reviewer Recommendation Console',
-            'instructions': '<p class="dark">Please select the reviewers you want to recommend for each paper.</p>\
-                <p class="dark"><strong>Please note:</strong></p>\
+            'title': conference.get_short_name() + ' Reviewer Recommendation',
+            'instructions': '<p class="dark">Recommend a ranked list of reviewer for each of your assigned papers.</p>\
+                <p class="dark"><strong>Instructions:</strong></p>\
                 <ul>\
-                    <li>The list of reviewers for each papers are sorted by assignment score.</li>\
-                    <li>Assigned: reviewers assigned using the first macthing, Alternate: next possible reviewers to assign.</li>\
-                </ul>\
-                <p class="dark"><strong>A few tips:</strong></p>\
-                <ul>\
-                    <li>.</li>\
-                    <li>.</li>\
+                    <li>Follow this link listed below to see your assigned papers.</li>\
+                    <li>For each of your assigned paper, select a ranked list of 7 reviewers.</li>\
```

Comment:
"For each or your assigned papers recommend 7 reviewers in a ranked list, where higher numbers represent stronger recommendations."